logverification/verifylist_test.go

```diff
@@ -58,3 +60,38 @@ func TestVerifyListIntegration(t *testing.T) {
 	//  in the given list of events, i.e. omitted events. We can call the datatrails
 	//  events api to list events filtered by mmr index.
 }
+
+func TestVerifyListOmmitted(t *testing.T) {
+	logger.New("TestVerifyList")
+	defer logger.OnExit()
+
+	testContext, testGenerator, _ := integrationsupport.NewAzuriteTestContext(t, "TestVerifyList")
+	tenantID := mmrtesting.DefaultGeneratorTenantIdentity
+	generatedEvents := integrationsupport.GenerateTenantLog(
+		&testContext, testGenerator, 8, tenantID, true, integrationsupport.TestMassifHeight,
+	)
+	trimmedGeneratedEvents := generatedEvents[1:]
+	eventJsonList := serializeTestEvents(t, trimmedGeneratedEvents)
+	omittedIndices, err := VerifyList(testContext.Storer, eventJsonList)
+
+	require.Nil(t, err)
+	require.Len(t, omittedIndices, 1)
```

Comment:
nice! we should have deterministic mmrIndexs on events as well, can we verify the mmr index returned from the omitted event is the one we expect?

Can we also do a list of > 1 for omitted events? I found a bug in the for loop where it wouldn't give more than 1 omitted event previously, want to ensure that case is captured.

Happy to have those tests as follow ups